content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -5,60 +5,87 @@ weight: 150
 
-It is strongly recommend against using your AWS root account for day-to-day tasks. Instead, create a user in IAM with programmatic access credentials. Best practice is to change this account's access key regularly and follow the practice of least privileges. For more information on managing access keys, see [Managing Access Keys for IAM Users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) in the AWS IAM User Guide.
+If you want to use short-term credentials for working with AWS, please see setup information in the [AWS Client Auth Gem](/docs/user-guide/gems/reference/aws/aws-client-auth).
 
-## AWS CLI
+Chose from the following options to set up a user's AWS credentials for use in O3DE. If you are using [named profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) remember to set the profile in the project settings.
 
-It is recommended to use the [AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) (CLI) (version 2) to manage the import and configuration of AWS credentials. If you have not configured credentials or a region on your computer, the easiest way to satisfy this requirement is to use the AWS `configure` command:
+### Option 1: Configure via the AWS CLI
+
+O3DE recommends using the [AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) (CLI) (version 2) to manage the import and configuration of AWS credentials. If you have not configured credentials or a region on your computer, the easiest way to satisfy this requirement is to use the AWS `configure` command:
 
 ```cmd
 aws configure
 ```
 
-Using this command you can provide your AWS access key ID, secret access key, and default region when prompted.
-
-If you don't have access keys and secret keys, you must sign in as an IAM user, create an access key, and download credentials using the steps shown in the AWS Reference Guide for [Programmatic Access](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys).
+Using this command you can provide your AWS access key ID, secret access key, and default region manually when prompted. 
 
-You can easily imported the exported credentials using the AWS `import` sub-command (requires AWS CLI version 2):
+Alternatively, when you create new access keys for a user, you are given the option to download the keys as a CSV. You can then automatically import them using the AWS `import` sub-command (requires AWS CLI version 2):
 
 ```cmd
 aws configure import --csv file://credentials.csv
```

Comment:
Nit: May worth a note here. When use the import command, the CLI doesn't configure the **default** profile but a new profile with the exact same name of the IAM user.